wpi_interface/src/test/java/unittest.java

```diff
@@ -0,0 +1,27 @@
+package coppercore.wpi_interface.test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.jupiter.api.Assertions.*;
+
+import coppercore.wpi_interface.Controllers;
+import java.util.List;
+// import org.junit.Before;
+import org.junit.jupiter.api.Test;
+
+public class unittest {
+
+    // @Before
+    // public static void TestPrep() {
+    //    Controllers.loadControllers();
+    // }
+
+    @Test
+    public static void TestOne() {
+        Controllers.loadControllers();
+        List<Controllers.Controller> controllers = Controllers.getControllers();
+        System.out.println(controllers.size());
+        assertEquals(23124, controllers.size());
+        assertFalse(true);
```

Comment:
Do these tests pass?

Reply:
Forgot to redo them I will that later today

Reply:
I was trying to get unit testing to work

Reply:
Oh yeah I remember they weren't running before right?

Reply:
Did you get unit tests working for the monitors?

Reply:
Yeah I got them to run as far as I remember